examples/cursors.py

```diff
@@ -1,104 +1,221 @@
+    cursors = [system_cursor1, system_cursor3, color_cursor, bitmap_cursor1, system_cursor2, bitmap_cursor2, image_cursor]
+    index = 0
+    pg.mouse.set_cursor(cursors[index])
+
+    while True:
+
+        mouse_pos = pg.mouse.get_pos
+        mouse_x = mouse_pos()[0]
+        mouse_y = mouse_pos()[1]
+
+        #Check if mouse is inside a circle to change its color
+        if check_circle(mouse_x, mouse_y, circle1.centerx, circle1.centery, radius1):
+            circle1 = pg.draw.circle(bg, (255, 0, 0), (pos_x1, pos_y1), radius1)
+        else:
+            circle1 = pg.draw.circle(bg, (255, 255, 255), (pos_x1, pos_y1), radius1)
+
+        if check_circle(mouse_x, mouse_y, circle2.centerx, circle2.centery, radius2):
+            circle2 = pg.draw.circle(bg, (255, 127, 0), (pos_x2, pos_y2), radius2)
+        else:
+            circle2 = pg.draw.circle(bg, (255, 255, 255), (pos_x2, pos_y2), radius2)
+
+        if check_circle(mouse_x, mouse_y, circle3.centerx, circle3.centery, radius3):
+            circle3 = pg.draw.circle(bg, (255, 255, 0), (pos_x3, pos_y3), radius3)
+        else:
+            circle3 = pg.draw.circle(bg, (255, 255, 255), (pos_x3, pos_y3), radius3)
+
+        if check_circle(mouse_x, mouse_y, circle4.centerx, circle4.centery, radius3):
+            circle4 = pg.draw.circle(bg, (0, 255, 0), (pos_x4, pos_y4), radius4)
+        else:
+            circle4 = pg.draw.circle(bg, (255, 255, 255), (pos_x4, pos_y4), radius4)
+
+        if check_circle(mouse_x, mouse_y, circle5.centerx, circle5.centery, radius4):
+            circle5 = pg.draw.circle(bg, (0, 0, 255), (pos_x5, pos_y5), radius5)
+        else:
+            circle5 = pg.draw.circle(bg, (255, 255, 255), (pos_x5, pos_y5), radius5)
+      
+        if check_circle(mouse_x, mouse_y, circle6.centerx, circle6.centery, radius6):
+            circle6 = pg.draw.circle(bg, (75, 0, 130), (pos_x6, pos_y6), radius6)
+        else:
+            circle6 = pg.draw.circle(bg, (255, 255, 255), (pos_x6, pos_y6), radius6)
+
+        if check_circle(mouse_x, mouse_y, circle7.centerx, circle7.centery, radius7):
+            circle7 = pg.draw.circle(bg, (148, 0, 211), (pos_x7, pos_y7), radius7)
+        else:
+            circle7 = pg.draw.circle(bg, (255, 255, 255), (pos_x7, pos_y7), radius7)
+
+    
+
+        bg.fill((183, 201, 226), (0, 15 ,bg.get_width(), 50))
+        text1 = font.render(("This is a "+ pg.mouse.get_cursor().type + " cursor"), True, (0, 0, 0))
+        text_rect1 = text1.get_rect(center=(bg.get_width()/2, 40))
+        bg.blit(text1, text_rect1)
+        
+        
+        for event in pg.event.get():
+
+            button = pg.draw.rect(bg, (100,149,240),(139, 300, button_text.get_width()+5, button_text.get_height() + 50))
+            bg.blit(button_text, button_text_rect)
+
+            #Check if button was clicked and change cursor
+            if mouse_x > button.x and mouse_x < button.x + button.width and mouse_y > button.y and mouse_y < button.y + button.height:
```

Comment:
we can rewrite this as:

```
if ( button.x < mouse_x < button.x + button.width and button.y < mouse_y < button.y + button.height ):
```

Reply:
Rect.collidepoint would probably be better, if I understand what this code is doing properly.

Reply:
Agreed.